File: C_Programming/Study/Q1_Assignment12.c

///////////////////////////////////////////////////////////////////////////////////////////////////////////
//
// Function Name :  DisplayDigit
// Output:          Display Digits from that Number
// Description:     Assignment_12: Q.1  : Write a program which accept Number from user & Display in Reverse Order.  
// Date:            30/05/2026
// Author:          Shital Ajit Nikam   
//
///////////////////////////////////////////////////////////////////////////////////////////////////////////

#include <stdio.h>

void DisplayDigit(int iNo)
{
    int iDigit = 0;
    int iCnt = 0;

    //Updator used

    if(iNo < 0)
    {
        iNo = -iNo;
    }
               
    while(iNo != 0)
    {
        iDigit = iNo % 10;
        printf("%d\n",iDigit);
        iNo = iNo /10;
    }
    
}

int main()
{
    int iValue = 0;
    int iRet = 0;
        
    printf("Enter Number: \n");
    scanf("%d",&iValue);

    DisplayDigit(iValue);
    
    return 0;
}

/////////////////////////////////////////////////////////////////////////////////////
//Time Complexity : O(N)  where N = length of Number
/////////////////////////////////////////////////////////////////////////////////////
// Input : 2395
// Output : 5   9   3   2
// Input : 1018
// Output : 8   1   0   1
// Input : -1018
// Output : 8   1   0   1 
// Input : 9000
// Output :0   0   0   9
// 
/////////////////////////////////////////////////////////////////////////////////////

File: Python_Programming/Competative/Q1_Assignment12.py
#///////////////////////////////////////////////////////////////////////////////////////////////////////////////////
# Function Name :  ChkVowel
#        
# Description:     Assignment_12: Q.1: Write a program which accepts one character & checks whether it is Vowel 
#                                      or Consonant.   
# Input  :         A
# Output :         It is Vowel
#                                     
# Date:            24/05/2026
# Author:          Shital Ajit Nikam   
#
#///////////////////////////////////////////////////////////////////////////////////////////////////////////////////

def ChkVowel(Ch):
    if(Ch == 'A' or Ch == 'E' or Ch == 'I' or Ch == 'O' or Ch == 'U' or Ch == 'a' or Ch == 'e' or Ch == 'i' or Ch == 'o' or Ch == 'u'):
        return True
    else :
        return False    
               
    
def main():
    value = input("Enter the Character: ")
        
    Ret = ChkVowel(value) 
    
    if(Ret == True):
        print("Character is Vowel")
    else:
        print("Character is Consonant")
              
    
if __name__ == "__main__":
    main()